### PR TITLE
Move "generic attributes" feature into C# 11

### PR DESCRIPTION
**Pull request by jcouv**

### CODE REVIEW

File: src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests.cs [hunk 1]
```diff
@@ -5806,9 +5806,9 @@ class Goo<T>
                 // (2,2): error CS0616: 'Goo<T>' is not an attribute class
                 // [Goo<int>]
                 Diagnostic(ErrorCode.ERR_NotAnAttributeClass, "Goo<int>").WithArguments("Goo<T>").WithLocation(2, 2),
-                // (2,2): error CS8652: The feature 'generic attributes' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                // (2,2): error CS8773: Feature 'generic attributes' is not available in C# 9.0. Please use language version 11.0 or greater.
```
Comment (333fred) [inline src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests.cs:5809]: Seems like most of these tests should move to C# 10.

**Review by 333fred — COMMENTED** [review]
Done review pass (commit 1)

**Review by 333fred — APPROVED** [review]
LGTM (commit 2)

**Comment by jcouv** [comment]
@dotnet/roslyn-compiler for second review. Thanks

**Review by cston — APPROVED** [review]

**Pull request merged**